src/mbgl/sprite/sprite_atlas.cpp

```diff
@@ -187,8 +187,15 @@ void SpriteAtlas::updateDirty() {
             // The two names match;
             Holder& holder = imageIterator->second;
             holder.texture = spriteIterator->second;
-            copy(holder, imageIterator->first.second);
-
+            if (holder.texture != nullptr)
+            {
```

Comment:
Nit: The C++ side of this project places braces on the same line, in contrast to the prevailing Objective-C style around here. (I’d be in favor of changing the Objective-C style to same-line braces for consistency.)